libs/utils/src/lib/command-runner.service.ts

```diff
@@ -95,6 +101,21 @@ export class CommandRunner {
               return of(c);
             }
           }),
+          distinctUntilChanged((a, b) => {
```

Comment:
I found that new value was being emitted by the observable due to polling (even though nothing has changed), which is causing the UI to thrash.

Reply:
Since the UI should consume chunks, you can get the same looking chunk multiple times in a row, each of which will need to be applied to the terminal.

Reply:
If we didn't use chunks, then it would have worked.